csharp/ql/integration-tests/all-platforms/cshtml/global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can't we put a single `global.json` file to `csharp/ql/integration-tests/`?

The docs say:
> The .NET SDK looks for a global.json file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.

Reply:
Seems like that would be worth trying to see if it works as expected. That would mean we'd only need to update the version in one place since we can point `setup-dotnet` at that file as well (see https://github.com/actions/setup-dotnet#using-the-global-json-file-input) if we choose to add an explicit step for that.

Reply:
I think our workflow generator already adds such a step.

Reply:
I will try and add it directly to the integration test directory and see if it works (there might be some issues with relying on the upwards search as the integration tests are copied before executed and I don't know what happens to the content of the parent folder (the `integration-tests` directory itself)).
Will also take a look at the workflow generation script and see.

Reply:
We will not be able to use a `global.json` file for all workflow descriptions. Should we put in an effort to use the `global.json` files in some cases? If we should do so it will be a PR in `semmle-code` and not anything we can do as a part of this PR.

Reply:
As discussed in person, there's no need to spend time on changing the workflow generator and the build system. I think we can merge this PR as it is.

Reply:
> > We will not be able to use a global.json file for all workflow descriptions.
> 
> What's the reason for that? A limitation of the workflow generator or something else?

Perhaps an overstatement, but it will not be easy to always use a `global.json` file:
The workflow generator adds the installation of the dotnet to all workflows, including workflows that builds the extractor and CLI, but maybe also some where it is not strictly required. That is, we need to figure out exactly which workflows that needs .NET and supply a `global.json` for those particular workflows (and maybe also re-write the workflow generator code to not add a `setup-dotnet` in some cases). This will go a bit beyond adding `global.json` files to the integration tests.

Reply:
@tamasvajk and I also discussed in person that it is probably for the best to keep the `global.json` file as one file pr. test (in the test folder) as there is some copying going on (that may or may not cover parent directories) when tests are executed, which might also differ locally and on-server and might behave differently when executing a single test <-- To be one safe side then we stick to one `global.json` file pr. test.